MANIFEST.in

```diff
@@ -1,4 +1,3 @@
 include README.rst
 graft gcloud
 global-exclude *.pyc
-recursive-exclude system_tests *
```

Comment:
For posterity, this is no longer necessary since `system_tests/__init__.py` is gone